internal/view/helpers.go

```diff
@@ -214,3 +215,24 @@ func decorateCpuMemHeaderRows(app *App, data *render.TableData) *render.TableDat
 
 	return data
 }
+
+func clipboardWrite(text string) error {
+	if err := clipboard.Init(); err != nil {
+		return err
+	}
+	if clipboard.Write(clipboard.FmtText, []byte(text)) == nil {
+		return errors.New("unable to write to clipboard")
+	}
+	return nil
+}
+
+func cpCmd(flash *model.Flash, text string) func(*tcell.EventKey) *tcell.EventKey {
```

Comment:
@mikutas I think best if you pass in the textview here vs text then you can just call cpCmd directly in the action mapper in the view.